### PR TITLE
Rename some functions

### PR DESCRIPTION
**Pull request by SoongNoonien**
I've renamed some methods as suggested in #41 and #185.

### CODE REVIEW

File: src/Ortho.jl [hunk 1]
```diff
@@ -78,20 +78,20 @@ function ortho2norm(t::CharTable, class::Int64)
...
 julia> g=greenfuntab("GL3");
 
-julia> ortho2norm(g,2)
+julia> norm(g,2)
 (5*q^2 + 2*q + 3)//(q^5 - 2*q^4 + q^3)
 ```
 """
-function ortho2norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
-	if class > classtypes(t)
+function Oscar.norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
```
Comment (fingolfin) [inline src/Ortho.jl:93]: We should just `import Oscar: norm` in a suitable place and then do
```suggestion
function norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
```

File: docs/src/classes.md [hunk 2]
```diff
@@ -30,7 +31,10 @@ satisfied if the evaluation of this polynomial is an integer.
...
 ```@docs
-ortho2norm
-ortho2scalar
-classmult
+norm(t::CharTable, class::Int64)
+norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly
+scalar_product(t::CharTable, class1::Int64, class2::Int64)
+scalar_product(t::SimpleCharTable{T}, class1::Int64, class2::Int64) where T <: NfPoly
+class_multiplication_coefficient(t::CharTable, class1::Int64, class2::Int64, class3::Int64)
+class_multiplication_coefficient(t::SimpleCharTable{T}, class1::Int64, class2::Int64, class3::Int64) where T <: NfPoly
```
Comment (fingolfin) [inline docs/src/classes.md:39]: I *think* you can write this as
```suggestion
norm(t::Table, class::Int64)
scalar_product(t:: Table, class1::Int64, class2::Int64)
class_multiplication_coefficient(t:: Table, class1::Int64, class2::Int64, class3::Int64)
```

File: src/Show.jl [hunk 3]
```diff
@@ -165,41 +181,41 @@ q^4 - q^3 - q^2 + q
...
 ```
 """
-function nrclasses(t::CharTable, class::Int64)
+function Oscar.number_of_conjugacy_classes(t::CharTable, class::Int64)
 	o=t.ring(1)
 	result=t.classsums[class](o//o)
 	return shrink(result)
 end
-nrclasses(t::SimpleCharTable, class::Int64) = t.classtypeorder[class]
+Oscar.number_of_conjugacy_classes(t::SimpleCharTable, class::Int64) = t.classtypeorder[class]
```
Comment (fingolfin) [inline src/Show.jl:218]: similar to above: just `import` this

**Review by fingolfin — APPROVED** [review]

File: docs/src/classes.md (hunk 2 above)
Comment (fingolfin) [reply docs/src/classes.md:39]: (This is of course not important. From my POV you can merge this as-is as soon as the tests passed)

**Comment by fingolfin** [comment]
```
ℤWARNING: both Oscar and GenericCharacterTables export "class_multiplication_coefficient"; uses of it in module Main must be qualified
│    ERROR: UndefVarError: `class_multiplication_coefficient` not defined
```

That suggests a missing `import Oscar: class_multiplication_coefficient`, maybe?

**Comment by fingolfin** [comment]
Perhaps time to also add `src/Imports.jl`, as in our other packages

File: src/Show.jl [hunk 4]
```diff
@@ -86,60 +84,76 @@ function Base.show(io::IO, ::MIME"text/plain", c::AbstractGenericCharacter)
...
+
+# Examples
+```jldoctest
+julia> g=genchartab("GL2");
+
+julia> number_of_character_types(g)
+4
+
+"""
+number_of_character_types(t::Table) = length(t)
```
Comment (SoongNoonien) [inline src/Show.jl:121]: Why is Codecov complaining here? There is a test in line 117.

**Comment by SoongNoonien** [comment]
I'll go ahead and merge this now and make a follow up PR which improves the code coverage in general. We are still lacking some tests for the simple tables.

File: src/Show.jl (hunk 4 above)
Comment (SoongNoonien) [reply src/Show.jl:121]: Ok, I missed some back-ticks in line 119...

**Pull request merged**